lib/stack_master/aws_driver/cloud_formation.rb

```diff
@@ -3,9 +3,15 @@ module AwsDriver
     class CloudFormation
       extend Forwardable
 
-      def set_region(region)
-        @region = region
-        @cf = nil
+      def region
+        @region ||= ENV['AWS_REGION'] || Aws.config[:region] || Aws.shared_config.region
+      end
+
+      def set_region(value)
+        if region != value
+          @region = value
+          @cf = nil
+        end
```

Comment:
@samuelsuarezsanchez 💯  on the explanation. Much appreciated!

IMHO, just leave it as is!

Awesome job 👏